src/components/Display.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+class Display extends Component {
+  static propTypes = {
+    result: PropTypes.string
+  };
+  render() {
+    return (
+      <div className="display-component">
+        <div>{this.props.result}</div>
+      </div>
```

Comment:
Use `propTypes` to ensure that `result` is a string
also use `defaultProps` to ensure that default value is '0'